docs/topics/proc_releasing-launcher-docs.adoc

```diff
@@ -39,7 +39,37 @@ Replace `$REMOTE` with the name of the upstream remote.
 --
 
 . Request publication:
-.. File a pull request in the link:https://github.com/openshiftio/saas-launchpad/blob/master/launchpad-services/launcher-documentation.yaml#L2[openshiftio/saas-launchpad] repository, where you change the `hash` value to the hash of the commit you want to publish from the link:{link-repo-docs}[{name-repo-docs}] repository. Usually, this will be the latest commit in the `master` branch.
+.. Fork link:https://access.redhat.com/documentation/en-us/red_hat_jboss_web_server/5.0/html/red_hat_jboss_web_server_for_openshift/index[openshiftio/saas-launchpad].
+.. Clone your fork:
++
+[source,bash]
+----
+git clone git@github.com/$GIT_USER/openshiftio/saas-launchpad.git
+----
+Replace `$GIT_USER` with your Git user name.
+
+.. Add the repository as a remote on your fork:
++
+[source,bash]
+----
+git remote add upstream git@github.com:$GIT_USER/openshiftio/saas-launchpad.git
+----
+.. Fetch upstream.
+.. Create a topic branch:
++
+[source,bash]
+----
+git checkout -v $BRANCH_NAME upstream/master
+----
+.. Edit the SHA in `launcher-documentation.yaml` on the topic branch and push your change to origin.
++
+The SHA value is from the commit you want to publish from the link:{link-repo-docs}[{name-repo-docs}] repository. Usually, this is the latest commit in the `master` branch.
++
+[source,bash]
+----
+$ git push origin/$BRANCH_NAME
+----
+.. File a pull request in the link:https://github.com/openshiftio/saas-launchpad/blob/master/launchpad-services/launcher-documentation.yaml#L2[openshiftio/saas-launchpad] repository.
```

Comment:
```suggestion
.. File a pull request in the link:https://github.com/openshiftio/saas-launchpad[openshiftio/saas-launchpad] repository.
```

Reply:
Would have taken this fix too if not for that pesky #L2 :-)